app/hotels/src/filter/hotelFacilities/HotelFacilitiesPopup.js

```diff
@@ -124,11 +126,13 @@ export default class HotelFacilitiesPopup extends React.Component<
           key={i}
           isChecked={facilities.indexOf(key) >= 0}
           onPress={this.handleCheckboxOnPress(key)}
-          style={i < facilitiesListKeys.length - 1 && styles.delimiter}
         >
           <View style={styles.checkbox}>
-            <Icon name={facility.icon} size={20} style={styles.facilityIcon} />
+            <Icon name={facility.icon} size={24} style={styles.facilityIcon} />
```

Comment:
Please try to use icons from https://orbit.kiwi/visual-style/icons/

they can be used like

```
import { TextIcon } from '@kiwicom/mobile-shared';
<TextIcon code="&#xe001;" orbit={true} />
```